python/unitytrainers/ppo/trainer.py

```diff
@@ -345,7 +390,7 @@ def update_model(self):
         """
         num_epoch = self.trainer_parameters['num_epoch']
         n_sequences = max(int(self.trainer_parameters['batch_size'] / self.sequence_length), 1)
-        total_v, total_p = [], []
+        total_v, total_p, total_f, total_i = [], [], [], []
```

Comment:
Maybe it is time to find better names for these variables or is this outside of the scope of this PR ?

Reply:
I will make them less terrible here. It is justified since we are adding new ones.